### PR TITLE
INFP-2431

### PR DESCRIPTION
**Pull request by dungnh165877**

### CODE REVIEW

File: case-studies/policy/README.md [hunk 1]
```diff
@@ -85,17 +85,23 @@ sudo apy-get install ipset
 
 # 2. Setup environment or .env file
 # api-server address
-API_SERVER_ADDRESS="http://localhost:8080"
+API_SERVER_ADDRESS="address of api-server"
 # ip address of api-server
-API_SERVER_IPV4="127.0.0.1"
+API_SERVER_IPV4="ipv4 of api-server"
```
Comment (quantonganh) [inline case-studies/policy/README.md:90]: What is the different between this one and API_SERVER_ADDRESS?

File: case-studies/policy/cli.md [hunk 2]
```diff
@@ -43,81 +45,148 @@ The create command is used to create resources by filename.
...
 
+## List resource
+```shell
+List resource
+
+Usage:
+  bbfw list [flags]
+
+Examples:
+  # List global network policy
```
Comment (quantonganh) [inline case-studies/policy/cli.md]: "global network service"?

File: case-studies/policy/cli.md [hunk 3]
```diff
@@ -43,81 +45,148 @@ The create command is used to create resources by filename.
...
+  bbfw list gns
+
+  # List global network policy
+  bbfw list gnp
+
+  # List global network policy with order
+  bbfw list gnp --isOrder
+
+  # List host endpoint
+  bbfw list hep
```
Comment (quantonganh) [inline case-studies/policy/cli.md:80]: "hep" is not commonly used to represent "host endpoints".

File: case-studies/policy/cli.md [hunk 4]
```diff
@@ -43,81 +45,148 @@ The create command is used to create resources by filename.
...
+  # Delete many policy with name
+  bbfw delete gnp allow_ssh allow_ping
+
+  # Delete many policy with filename
+  bbfw delete gnp -f allow_ssh.yaml -f allow_ping.yaml
+
+  # Delete a set with name
+  bbfw delete gns server
 
   # Delete many policy with name
```
Comment (quantonganh) [inline case-studies/policy/cli.md]: "many sets".

File: case-studies/policy/cli.md [hunk 5]
```diff
@@ -43,81 +45,148 @@ The create command is used to create resources by filename.
...
-  bbfwcli delete hep allow_ssh allow_ping
+  bbfw delete gns server vm
 
   # Delete many policy with filename
```
Comment (quantonganh) [inline case-studies/policy/cli.md]: "many sets".

File: case-studies/policy/cli.md [hunk 6]
```diff
@@ -43,81 +45,148 @@ The create command is used to create resources by filename.
...
-  bbfwcli delete hep allow_ssh.yaml allow_ping.yaml
+  bbfw delete gns -f server.yaml -f vm.yaml
+
+  # Delete a hep with tenantID and ip
+  bbfw delete hep --tenantID=1 --ip=192.168.1.1
```
Comment (quantonganh) [inline case-studies/policy/cli.md:164]: I often misread "hep" as "help". What do you think if we use "endpoints"?

File: case-studies/policy/cli.md [hunk 7]
```diff
@@ -43,81 +45,148 @@ The create command is used to create resources by filename.
...
+
+  # Delete many hep with filename
```
Comment (quantonganh) [inline case-studies/policy/cli.md]: "many endpoints".

File: case-studies/policy/cli.md [hunk 8]
```diff
@@ -43,81 +45,148 @@ The create command is used to create resources by filename.
...
 Flags:
   -f, --file stringArray   file to read
   -h, --help               help for delete
+      --ip string          HEP: get by ip
+      --tenantID uint      HEP: get by tenantID
 ```
 
 ## Auto completion command
 ```shell
 Generate bash completion script for shell(bash, zsh)
```
Comment (quantonganh) [inline case-studies/policy/cli.md]: Generate a completion script for bash or zsh shell.

File: case-studies/policy/README.md (hunk 1 above)
Comment (dungnh165877) [reply case-studies/policy/README.md:90]: API_SERVER_ADDRESS for agent call API. API_SERVER_ADDRESS is domain name or ip with port
API_SERVER_IPV4 is ip of api-server. Use for iptables.
I have add more comments

File: case-studies/policy/cli.md (hunk 6 above)
Comment (dungnh165877) [reply case-studies/policy/cli.md:164]: if use endpoints, we will confuse endpoints for what. hep is abbreviation of hostendpoint. You can use bbfw delete hostendpoint

File: case-studies/policy/cli.md (hunk 3 above)
Comment (dungnh165877) [reply case-studies/policy/cli.md:80]: hep is abbreviation of hostendpoint. You can use bbfw list hostendpoint

**Comment by quantonganh** [comment]
LGTM.

**Review by bienkma — APPROVED** [review]

**Pull request merged**